synapse/storage/stream.py

```diff
@@ -738,38 +678,49 @@ def update_federation_out_pos(self, typ, stream_id):
     def has_room_changed_since(self, room_id, stream_id):
         return self._events_stream_cache.has_entity_changed(room_id, stream_id)
 
+    def paginate_room_events_txn(self, txn, room_id, from_token, to_token=None,
```

Comment:
Is there any change in functionality here, or is it just:

* move paginate_room_events to StreamWorkerStore instead of StreamStore
* pull paginate_room_events_txn out to a member function rather than being a nested `f` within paginate_room_events
* do some of the stuff that was in paginate_room_events in paginate_room_events_txn?

I think it's the latter but I'm not sure if I'm missing something.

Reply:
Yup, there shouldn't be any functional change here.

(I really wish GH made it clearer that stuff is mainly white space change, e.g. https://github.com/matrix-org/synapse/pull/3193/commits/06c0d0ed081b56716135c4881e600861b2e8cad5?w=1 makes this a lot clearer imo)